core/src/AzureMcp.Core/Commands/GlobalCommand.cs

```diff
@@ -5,6 +5,8 @@
 using Azure;
 using Azure.Core;
 using Azure.Identity;
+using Microsoft.Mcp.Core.Commands;
```

Comment:
This using statement is duplicated.  Neither of them may be needed as it looks like you're using the fully qualified type name below.